Islandora-CLAW/islandora
# Use standard JWT claims

**GitHub Issue**: https://github.com/Islandora-CLAW/CLAW/issues/736

Related to: https://github.com/Islandora-CLAW/Syn/pull/11
Related to: https://github.com/Islandora-CLAW/Crayfish-Commons/pull/15

# What does this Pull Request do?

Changes to the provided claims in the JWT.

# What's new?

* Does this change require documentation to be updated? Might be good to document the JWT claims somewhere, eventually.
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository
 (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

See testing instructions here (https://github.com/Islandora-CLAW/Syn/pull/11) this pulls in the changes to Syn, islandora and Crayfish-Commons.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora-CLAW/committers
